<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Referencias | Evaluación de métricas para la búsqueda de biomarcadores de diagnóstico y pronóstico clínico</title>
  <meta name="description" content="Referencias | Evaluación de métricas para la búsqueda de biomarcadores de diagnóstico y pronóstico clínico" />
  <meta name="generator" content="bookdown 0.40 and GitBook 2.6.7" />

  <meta property="og:title" content="Referencias | Evaluación de métricas para la búsqueda de biomarcadores de diagnóstico y pronóstico clínico" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Referencias | Evaluación de métricas para la búsqueda de biomarcadores de diagnóstico y pronóstico clínico" />
  
  
  

<meta name="author" content="Pablo Navarro" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="session-info.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.33/datatables.js"></script>
<link href="libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>


<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path=""><a href="#introducci%C3%B3n"><i class="fa fa-check"></i>Introducción</a>
<ul>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html"><i class="fa fa-check"></i>Conjuntos de datos</a></li>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html#software-utilizado"><i class="fa fa-check"></i>Software utilizado</a>
<ul>
<li class="chapter" data-level="" data-path=""><a href="#librer%C3%ADas-externas"><i class="fa fa-check"></i>Librerías externas</a></li>
<li class="chapter" data-level="" data-path=""><a href="#librer%C3%ADas-propias"><i class="fa fa-check"></i>Librerías propias</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html#estructura"><i class="fa fa-check"></i>Estructura</a></li>
</ul></li>
<li class="part"><span><b>I Búsqueda de biomarcadores</b></span></li>
<li class="chapter" data-level="1" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html"><i class="fa fa-check"></i><b>1</b> Selección de células luminales de prostata</a>
<ul>
<li class="chapter" data-level="1.1" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#importar-matrices-de-cuentas"><i class="fa fa-check"></i><b>1.1</b> Importar matrices de cuentas</a></li>
<li class="chapter" data-level="1.2" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#preprocesamiento-y-cq"><i class="fa fa-check"></i><b>1.2</b> Preprocesamiento y CQ</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="07_References.html"><a href="#creaci%C3%B3n-del-objeto-seurat"><i class="fa fa-check"></i><b>1.2.1</b> Creación del objeto Seurat</a></li>
<li class="chapter" data-level="1.2.2" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#lecturas-mapeadas-al-genoma-mitocondrial"><i class="fa fa-check"></i><b>1.2.2</b> Lecturas mapeadas al genoma mitocondrial</a></li>
<li class="chapter" data-level="1.2.3" data-path="07_References.html"><a href="#normalizaci%C3%B3n"><i class="fa fa-check"></i><b>1.2.3</b> Normalización</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="07_References.html"><a href="#b%C3%BAsqueda-de-genes-de-alta-variabilidad"><i class="fa fa-check"></i><b>1.3</b> Búsqueda de genes de alta variabilidad</a></li>
<li class="chapter" data-level="1.4" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#escalado-de-los-datos"><i class="fa fa-check"></i><b>1.4</b> Escalado de los datos</a></li>
<li class="chapter" data-level="1.5" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#dim-reduction"><i class="fa fa-check"></i><b>1.5</b> Reducción de la dimensionalidad</a></li>
<li class="chapter" data-level="1.6" data-path="07_References.html"><a href="#integraci%C3%B3n-de-datos"><i class="fa fa-check"></i><b>1.6</b> Integración de datos</a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="07_References.html"><a href="#objetivos-de-la-integraci%C3%B3n"><i class="fa fa-check"></i><b>1.6.1</b> Objetivos de la integración</a></li>
<li class="chapter" data-level="1.6.2" data-path="07_References.html"><a href="#integraci%C3%B3n"><i class="fa fa-check"></i><b>1.6.2</b> Integración</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#dimensionalidad-del-conjunto-de-datos"><i class="fa fa-check"></i><b>1.7</b> Dimensionalidad del conjunto de datos</a></li>
<li class="chapter" data-level="1.8" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#clustering"><i class="fa fa-check"></i><b>1.8</b> Agrupamiento de células</a></li>
<li class="chapter" data-level="1.9" data-path="07_References.html"><a href="#reducci%C3%B3n-de-la-dimensionalidad-no-lineal-umap"><i class="fa fa-check"></i><b>1.9</b> Reducción de la dimensionalidad no lineal (UMAP)</a>
<ul>
<li class="chapter" data-level="1.9.1" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#utilidad-del-umap"><i class="fa fa-check"></i><b>1.9.1</b> Utilidad del UMAP</a></li>
<li class="chapter" data-level="1.9.2" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#umap"><i class="fa fa-check"></i><b>1.9.2</b> UMAP</a></li>
<li class="chapter" data-level="1.9.3" data-path="07_References.html"><a href="#visualizaci%C3%B3n-del-umap"><i class="fa fa-check"></i><b>1.9.3</b> Visualización del UMAP</a></li>
</ul></li>
<li class="chapter" data-level="1.10" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#degs-search"><i class="fa fa-check"></i><b>1.10</b> Búsqueda de genes expresados diferencialmente (DEG)</a></li>
<li class="chapter" data-level="1.11" data-path="07_References.html"><a href="#selecci%C3%B3n-de-c%C3%A9lulas"><i class="fa fa-check"></i><b>1.11</b> Selección de células</a>
<ul>
<li class="chapter" data-level="1.11.1" data-path="luminal-cells-selection.html"><a href="luminal-cells-selection.html#filtrado-de-resultados-significativos"><i class="fa fa-check"></i><b>1.11.1</b> Filtrado de resultados significativos</a></li>
<li class="chapter" data-level="1.11.2" data-path="07_References.html"><a href="#identificaci%C3%B3n-de-clusters"><i class="fa fa-check"></i><b>1.11.2</b> Identificación de clusters</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-cells-selection.html"><a href="cancer-cells-selection.html"><i class="fa fa-check"></i><b>2</b> Selección de células cancerosas y marcadores</a>
<ul>
<li class="chapter" data-level="2.1" data-path="cancer-cells-selection.html"><a href="cancer-cells-selection.html#deg-luminal-cells"><i class="fa fa-check"></i><b>2.1</b> DEG en células luminales</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="07_References.html"><a href="#caracterizaci%C3%B3n-y-selecci%C3%B3n-de-los-clusters"><i class="fa fa-check"></i><b>2.1.1</b> Caracterización y selección de los clusters</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="07_References.html"><a href="#b%C3%BAsqueda-de-subpoblaciones"><i class="fa fa-check"></i><b>2.2</b> Búsqueda de subpoblaciones</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="07_References.html"><a href="#repetici%C3%B3n-del-flujo-de-trabajo"><i class="fa fa-check"></i><b>2.2.1</b> Repetición del flujo de trabajo</a></li>
<li class="chapter" data-level="2.2.2" data-path="07_References.html"><a href="#interpretaci%C3%B3n-de-resultados"><i class="fa fa-check"></i><b>2.2.2</b> Interpretación de resultados</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="07_References.html"><a href="#selecci%C3%B3n-de-biomarcadores"><i class="fa fa-check"></i><b>2.3</b> Selección de biomarcadores</a></li>
</ul></li>
<li class="part"><span><b>II Generación del conjunto de datos</b></span></li>
<li class="chapter" data-level="3" data-path="count-matrix-generation.html"><a href="count-matrix-generation.html"><i class="fa fa-check"></i><b>3</b> Generación de la matriz de conteo</a>
<ul>
<li class="chapter" data-level="3.1" data-path="07_References.html"><a href="#descripci%C3%B3n-de-los-datos"><i class="fa fa-check"></i><b>3.1</b> Descripción de los datos</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="count-matrix-generation.html"><a href="count-matrix-generation.html#clinical-preprocessing"><i class="fa fa-check"></i><b>3.1.1</b> Preprocesamiento del fichero clinical y gdc_sample_sheet</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="07_References.html"><a href="#creaci%C3%B3n-de-matriz-de-conteo"><i class="fa fa-check"></i><b>3.2</b> Creación de matriz de conteo</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="count-matrix-generation.html"><a href="count-matrix-generation.html#import-read-files"><i class="fa fa-check"></i><b>3.2.1</b> Importar archivos de lecturas</a></li>
<li class="chapter" data-level="3.2.2" data-path="count-matrix-generation.html"><a href="count-matrix-generation.html#process-read-files"><i class="fa fa-check"></i><b>3.2.2</b> Selección de variables y renombrado</a></li>
<li class="chapter" data-level="3.2.3" data-path="07_References.html"><a href="#uni%C3%B3n-de-archivos"><i class="fa fa-check"></i><b>3.2.3</b> Unión de archivos</a></li>
<li class="chapter" data-level="3.2.4" data-path="07_References.html"><a href="#construcci%C3%B3n-de-la-matriz-de-conteo"><i class="fa fa-check"></i><b>3.2.4</b> Construcción de la matriz de conteo</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="count-matrix-generation.html"><a href="count-matrix-generation.html#procesado-de-la-matriz-de-conteo"><i class="fa fa-check"></i><b>3.3</b> Procesado de la matriz de conteo</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="count-matrix-generation.html"><a href="count-matrix-generation.html#pivotado-de-la-matriz"><i class="fa fa-check"></i><b>3.3.1</b> Pivotado de la matriz</a></li>
<li class="chapter" data-level="3.3.2" data-path="07_References.html"><a href="#c%C3%A1lculo-de-gold-standards"><i class="fa fa-check"></i><b>3.3.2</b> Cálculo de Gold Standards</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="count-matrix-filtering.html"><a href="count-matrix-filtering.html"><i class="fa fa-check"></i><b>4</b> Filtrado de la matriz de conteo</a>
<ul>
<li class="chapter" data-level="4.1" data-path="count-matrix-filtering.html"><a href="count-matrix-filtering.html#equivalencias-entre-identificadores"><i class="fa fa-check"></i><b>4.1</b> Equivalencias entre identificadores</a></li>
<li class="chapter" data-level="4.2" data-path="count-matrix-filtering.html"><a href="count-matrix-filtering.html#filtrado-por-los-genes-identificados"><i class="fa fa-check"></i><b>4.2</b> Filtrado por los genes identificados</a></li>
</ul></li>
<li class="part"><span><b>III Evaluación de índices</b></span></li>
<li class="chapter" data-level="5" data-path="index-evaluation.html"><a href="index-evaluation.html"><i class="fa fa-check"></i><b>5</b> Evaluación de índices</a>
<ul>
<li class="chapter" data-level="5.1" data-path="index-evaluation.html"><a href="index-evaluation.html#aucs"><i class="fa fa-check"></i><b>5.1</b> AUCs</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="07_References.html"><a href="#introducci%C3%B3n-1"><i class="fa fa-check"></i><b>5.1.1</b> Introducción</a></li>
<li class="chapter" data-level="5.1.2" data-path="07_References.html"><a href="#c%C3%A1lculo-de-aucs-para-diagn%C3%B3stico"><i class="fa fa-check"></i><b>5.1.2</b> Cálculo de AUCs para diagnóstico</a></li>
<li class="chapter" data-level="5.1.3" data-path="07_References.html"><a href="#c%C3%A1lculo-de-aucs-para-pron%C3%B3stico"><i class="fa fa-check"></i><b>5.1.3</b> Cálculo de AUCs para pronóstico</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="index-evaluation.html"><a href="index-evaluation.html#pauc"><i class="fa fa-check"></i><b>5.2</b> pAUC</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="07_References.html"><a href="#introducci%C3%B3n-2"><i class="fa fa-check"></i><b>5.2.1</b> Introducción</a></li>
<li class="chapter" data-level="5.2.2" data-path="07_References.html"><a href="#c%C3%A1lculo-de-paucs-en-regiones-de-alta-especificidad"><i class="fa fa-check"></i><b>5.2.2</b> Cálculo de pAUCs en regiones de alta especificidad</a></li>
<li class="chapter" data-level="5.2.3" data-path="07_References.html"><a href="#c%C3%A1lculo-de-paucs-en-regiones-de-alta-sensibilidad"><i class="fa fa-check"></i><b>5.2.3</b> Cálculo de pAUCs en regiones de alta sensibilidad</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="07_References.html"><a href="#%C3%ADndices-derivados-de-pauc"><i class="fa fa-check"></i><b>5.3</b> Índices derivados de pAUC</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="index-evaluation.html"><a href="index-evaluation.html#problemas-del-pauc"><i class="fa fa-check"></i><b>5.3.1</b> Problemas del pAUC</a></li>
<li class="chapter" data-level="5.3.2" data-path="07_References.html"><a href="#%C3%ADndices-para-la-especificidad"><i class="fa fa-check"></i><b>5.3.2</b> Índices para la especificidad</a></li>
<li class="chapter" data-level="5.3.3" data-path="07_References.html"><a href="#%C3%ADndices-para-la-sensibilidad"><i class="fa fa-check"></i><b>5.3.3</b> Índices para la sensibilidad</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="07_References.html"><a href="#comparaci%C3%B3n-de-%C3%ADndices"><i class="fa fa-check"></i><b>5.4</b> Comparación de índices</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="index-evaluation.html"><a href="index-evaluation.html#agrupar-por-condiciones"><i class="fa fa-check"></i><b>5.4.1</b> Agrupar por condiciones</a></li>
<li class="chapter" data-level="5.4.2" data-path="07_References.html"><a href="#biomarcadores-para-diagn%C3%B3stico"><i class="fa fa-check"></i><b>5.4.2</b> Biomarcadores para diagnóstico</a></li>
<li class="chapter" data-level="5.4.3" data-path="07_References.html"><a href="#biomarcadores-para-pron%C3%B3stico"><i class="fa fa-check"></i><b>5.4.3</b> Biomarcadores para pronóstico</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="index-evaluation.html"><a href="index-evaluation.html#resultados"><i class="fa fa-check"></i><b>5.5</b> Resultados</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="07_References.html"><a href="#rendimiento-de-m%C3%A9tricas"><i class="fa fa-check"></i><b>5.5.1</b> Rendimiento de métricas</a></li>
<li class="chapter" data-level="5.5.2" data-path="07_References.html"><a href="#relevancia-cl%C3%ADnica"><i class="fa fa-check"></i><b>5.5.2</b> Relevancia clínica</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="07_References.html"><a href="#discusi%C3%B3n-y-conclusiones"><i class="fa fa-check"></i><b>5.6</b> Discusión y conclusiones</a></li>
</ul></li>
<li class="appendix"><span><b>Apéndice</b></span></li>
<li class="chapter" data-level="A" data-path="session-info.html"><a href="session-info.html"><i class="fa fa-check"></i><b>A</b> Información de la sesión</a></li>
<li class="chapter" data-level="" data-path="referencias.html"><a href="referencias.html"><i class="fa fa-check"></i>Referencias</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Evaluación de métricas para la búsqueda de biomarcadores de diagnóstico y pronóstico clínico</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="referencias" class="section level1 unnumbered hasAnchor">
<h1>Referencias<a href="referencias.html#referencias" class="anchor-section" aria-label="Anchor link to header"></a></h1>

<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-esrp2-marker" class="csl-entry">
Advani, Rahul, Sara Luzzi, Emma Scott, Caroline Dalgliesh, Joachim Weischenfeldt, Jennifer Munkley, and David J. Elliot. 2023. <span>“Epithelial Specific Splicing Regulator Proteins as Emerging Oncogenes in Aggressive Prostate Cancer.”</span> <em>Oncogene</em> 42: 3161–68.
</div>
<div id="ref-osca" class="csl-entry">
Amezquita, Robert, Aaron Lun, Stephanie Hicks, Raphael Gottardo, and Alan O’Callaghan. 2024. <span>“Orchestrating Single-Cell Analysis with Bioconductor.”</span> In. Bioconductor. <a href="https://bioconductor.org/books/release/OSCA/">https://bioconductor.org/books/release/OSCA/</a>.
</div>
<div id="ref-luminal-markers" class="csl-entry">
Barclay, Wendy W., Linara S. Axanova, Wenhong Chen, Lina Romero, Sophia L. Maund, Shay Soker, Cynthia J. Lees, and Scott D. Cramer. 2008. <span>“Characterization of Adult Prostatic Progenitor/Stem Cells Exhibiting Self-Renewal and Multilineage Differentiation.”</span> <em>Stem Cells</em> 26 (3): 600–610.
</div>
<div id="ref-R-data.table" class="csl-entry">
Barrett, Tyson, Matt Dowle, Arun Srinivasan, Jan Gorecki, Michael Chirico, and Toby Hocking. 2024. <em>Data.table: Extension of ‘Data.frame‘</em>. <a href="https://r-datatable.com">https://r-datatable.com</a>.
</div>
<div id="ref-cxcl1-marker" class="csl-entry">
Benelli, Roberto, Sara Stigliani, Simona Minghelli, Sebastiano Carlone, and Nicoletta Ferrari. 2013. <span>“Impact of CXCL1 Overexpression on Growth and Invasion of Prostate Cancer Cell.”</span> <em>Prostate</em> 73 (9): 941–51.
</div>
<div id="ref-high-variability-genes" class="csl-entry">
Brennecke, Philip, Simon Anders, Jong Kyoung Kim, Aleksandra A. Kołodziejczyk, Xiuwei Zhang, Valentina Proserpio, Bianka Baying, et al. 2013. <span>“Accounting for Technical Noise in Single-Cell RNA-Seq Experiments.”</span> <em>Nat Methods</em>, 1093–95.
</div>
<div id="ref-dkk1-marker" class="csl-entry">
Browne, A. J., A. Göbel, S. Thiele, L. C. Hofbauer, M. Rauner, and T. D. Rachner. 2016. <span>“P38 MAPK Regulates the Wnt Inhibitor Dickkopf-1 in Osteotropic Prostate Cancer Cells.”</span> <em>Cell Death Dis.</em> 7 (2): 2119.
</div>
<div id="ref-agr2-marker" class="csl-entry">
Bu, Huajie, Stefanie Bormann, Georg Schäfer, Wolfgang Horninger, Petra Massoner, Antje Neeb, Vinoth-Kumar Lakshmanan, et al. 2011. <span>“The Anterior Gradient 2 (AGR2) Gene Is Overexpressed in Prostate Cancer and May Be Useful as a Urine Sediment Marker for Prostate Cancer Detection.”</span> <em>Prostate</em> 71 (6): 575–87.
</div>
<div id="ref-rab23-marker" class="csl-entry">
Chang, Junkai, Weibo Xu, Guangchao Liu, Xinyi Du, and Xiaodong Li. 2017. <span>“Downregulation of Rab23 in Prostate Cancer Inhibits Tumor Growth in Vitro and in Vivo.”</span> <em>Oncol. Res.</em> 25 (2): 241–48.
</div>
<div id="ref-best-practices" class="csl-entry">
Conesa, Ana, Pedro Madrigal, Sonia Tarazona, David Gomez-Cabrero, Alejandra Cervera, Andrew McPherson, Michał Wojciech Szcześniak, et al. 2016. <span>“A Survey of Best Practices for RNA-Seq Data Analysis.”</span> <em>Genome Biology</em> 17 (13).
</div>
<div id="ref-R-biomaRt" class="csl-entry">
Durinck, Steffen, and Wolfgang Huber. 2024. <em>biomaRt: Interface to BioMart Databases (i.e. Ensembl)</em>. <a href="https://doi.org/10.18129/B9.bioc.biomaRt">https://doi.org/10.18129/B9.bioc.biomaRt</a>.
</div>
<div id="ref-decipher" class="csl-entry">
Erho, Nicholas, Anamaria Crisan, Ismael A. Vergara, Anirban P. Mitra, Mercedeh Ghadessi, Christine Buerki, Eric J. Bergstralh, et al. 2013. <span>“Discovery and Validation of a Prostate Cancer Genomic Classifier That Predicts Early Metastasis Following Radical Prostatectomy.”</span> <em>PLoS One</em> 8 (6).
</div>
<div id="ref-risk-tests" class="csl-entry">
Farha, Mark W., and Simpa S. Salami. 2022. <span>“Biomarkers for Prostate Cancer Detection and Risk Stratification.”</span> <em>Ther Adv Urol.</em> 14.
</div>
<div id="ref-fpauc" class="csl-entry">
Franco, Manuel, and Juana-María Vivo. 2021. <span>“Evaluating the Performances of Biomarkers over a Restricted Domain of High Sensitivity.”</span> <em>Mathematics</em> 9 (21): 2826.
</div>
<div id="ref-R-ROCpAI" class="csl-entry">
Garcia, Juan-Pedro, Manuel Franco, and Juana-María Vivo. 2024. <em>ROCpAI: Receiver Operating Characteristic Partial Area Indexes for Evaluating Classifiers</em>.
</div>
<div id="ref-folh1-marker" class="csl-entry">
Haberkorn, Uwe, Matthias Eder, Klaus Kopka, John W. Babich, and Michael Eisenhub. 2016. <span>“New Strategies in Prostate Cancer: Prostate-Specific Membrane Antigen (PSMA) Ligands for Diagnosis and Therapy.”</span> <em>Clin Cancer Res.</em> 22 (1): 9–15.
</div>
<div id="ref-qc-sc" class="csl-entry">
Ilicic, Tomislav, Jong Kyoung Kim, Aleksandra A. Kolodziejczyk, Frederik Otzen Bagger, Davis James McCarthy, John C. Marioni, and Sarah A. Teichmann. 2016. <span>“Classification of Low Quality Cells from Single-Cell RNA-Seq Data.”</span> <em>Genome Biology</em> 17: 29.
</div>
<div id="ref-amacr-marker" class="csl-entry">
Jiang, Ning, Shimiao Zhu, Jing Chen, Yuanjie Niu, and Liqun Zhou. 2013. <span>“A-Methylacyl-CoA Racemase (AMACR) and Prostate-Cancer Risk: A Meta-Analysis of 4,385 Participants.”</span> <em>PLoS One</em> 8 (10).
</div>
<div id="ref-npauc" class="csl-entry">
Jiang, Yulei, Charles E. Metz, and Robert M. Nishikawa. 1996. <span>“A Receiver Operating Characteristic Partial Area Index for Highly Sensitive Diagnostic Tests.”</span> <em>Radiology</em> 201 (3): 745–50.
</div>
<div id="ref-oncotype" class="csl-entry">
Klein, Eric A., Matthew R. Cooperber, Cristina Magi-Galluzzi, Jeffry P. Simko, Sara M. Falzarano, Tara Maddala, June M. Chan, et al. 2014. <span>“A 17-Gene Assay to Predict Prostate Cancer Aggressiveness in the Context of Gleason Grade Heterogeneity, Tumor Multifocality, and Biopsy Undersampling.”</span> <em>Eur. Urol.</em> 66 (3): 550–60.
</div>
<div id="ref-pauc-comparison" class="csl-entry">
Ma, Hua, Andriy I. Bandos, and David Gur. 2015. <span>“On the Use of Partial Area Under the ROC Curve for Comparison of Two Diagnostic Tests.”</span> <em>Biometrical Journal</em> 57 (2): 304–20.
</div>
<div id="ref-base" class="csl-entry">
Ma, Xiaoshi, Jinanand Guo, Kaisheng Liu, Lipeng Chen, Dale Liu, Shaowei Dong, Jinquan Xia, et al. 2020. <span>“Identification of a Distinct Luminal Subgroup Diagnosing and Stratifying Early Stage Prostate Cancer by Tissue-Based Single-Cell RNA Sequencing.”</span> <em>Molecular Cancer</em> 19: 147.
</div>
<div id="ref-spauc" class="csl-entry">
McClish, Donna Katzman. 1989. <span>“Analyzing a Portion of the ROC Curve.”</span> <em>Medical Decision Making</em> 9 (3): 190–95.
</div>
<div id="ref-R-tibble" class="csl-entry">
Müller, Kirill, and Hadley Wickham. 2023. <em>Tibble: Simple Data Frames</em>. <a href="https://tibble.tidyverse.org/">https://tibble.tidyverse.org/</a>.
</div>
<div id="ref-R-pROC" class="csl-entry">
Robin, Xavier, Natacha Turck, Alexandre Hainard, Natalia Tiberti, Frédérique Lisacek, Jean-Charles Sanchez, and Markus Müller. 2023. <em>pROC: Display and Analyze ROC Curves</em>. <a href="https://xrobin.github.io/pROC/">https://xrobin.github.io/pROC/</a>.
</div>
<div id="ref-R-Seurat" class="csl-entry">
Satija, Rahul. 2024. <em>Seurat: Tools for Single Cell Genomics</em>. <a href="https://satijalab.org/seurat">https://satijalab.org/seurat</a>.
</div>
<div id="ref-lxn-marker" class="csl-entry">
Seed, Robert I., Alberto J. Taurozzi, Daniel J. Wilcock, Giovanna Nappo, Holger H. H. Erb, Martin L. Read, Mark Gurney, et al. 2019. <span>“The Putative Tumour Suppressor Protein Latexin Is Secreted by Prostate Luminal Cells and Is Downregulated in Malignancy.”</span> <em>Sci Rep.</em> 9 (1): 5120.
</div>
<div id="ref-folh1-therapeutic" class="csl-entry">
Sheehan, Beshara, Christina Guo, Antje Neeb, Alec Paschalis, Shahneen Sandhu, and Johann S. de Bono. 2022. <span>“Prostate-Specific Membrane Antigen Biology in Lethal Prostate Cancer and Its Therapeutic Implications.”</span> <em>European Urology Focus</em> 8 (5): 1157–68.
</div>
<div id="ref-integration" class="csl-entry">
Stuart, Tim, Andrew Butler, Paul Hoffman, Christoph Hafemeister, Efthymia Papalexi, William M. Mauck 3rd, Yuhan Hao, Marlon Stoeckius, Peter Smibert, and Rahul Satija. 2019. <span>“Comprehensive Integration of Single-Cell Data.”</span> <em>Cell</em> 177 (7): 1888–1902.
</div>
<div id="ref-slpi-marker" class="csl-entry">
Thompson, Maxwell, Jacques Lapoint, Yoon-La Choi, David E. Ong, John P. Higgins, James D. Brooks, and Jonathan R. Pollack. 2008. <span>“Identification of Candidate Prostate Cancer Genes Through Comparative Expression-Profiling of Seminal Vesicle.”</span> <em>Prostate</em> 68 (11): 1248–56.
</div>
<div id="ref-npy-marker" class="csl-entry">
Ueda, Koji, Ayako Tatsuguchi, Naomi Saichi, Atsuhiko Toyama, Kenji Tamura, Mutsuo Furihata, Ryo Takata, et al. 2013. <span>“Plasma Low-Molecular-Weight Proteome Profiling Identified Neuropeptide-y as a Prostate Cancer Biomarker Polypeptide.”</span> <em>J Proteome Res.</em> 12 (10): 4497–4506.
</div>
<div id="ref-tpauc" class="csl-entry">
Vivo, Juana-María, Manuel Franco, and Donatella Vicari. 2018. <span>“Rethinking an ROC Partial Area Index for Evaluating the Classification Performance at a High Specificity Range.”</span> <em>Advances in Data Analysis and Classification</em> 12: 683–704.
</div>
<div id="ref-early-detection" class="csl-entry">
Wang, Zhanfeng, and Yan-Chin Ivan Chang. 2011. <span>“Marker Selection via Maximizing the Partial Area Under the ROC Curve of Linear Risk Scores.”</span> <em>Biostatistics</em> 12 (2): 369–85.
</div>
<div id="ref-tumor-heterogeneity" class="csl-entry">
Wasim, Sobia, Sang-Yong Lee, and Jaehong Kim. 2022. <span>“Complexities of Prostate Cancer.”</span> <em>Int. J. Mol. Sci.</em> 23 (22): 14257.
</div>
<div id="ref-R-forcats" class="csl-entry">
Wickham, Hadley. 2023a. <em>Forcats: Tools for Working with Categorical Variables (Factors)</em>. <a href="https://forcats.tidyverse.org/">https://forcats.tidyverse.org/</a>.
</div>
<div id="ref-R-stringr" class="csl-entry">
———. 2023b. <em>Stringr: Simple, Consistent Wrappers for Common String Operations</em>. <a href="https://stringr.tidyverse.org">https://stringr.tidyverse.org</a>.
</div>
<div id="ref-R-dplyr" class="csl-entry">
Wickham, Hadley, Romain François, Lionel Henry, Kirill Müller, and Davis Vaughan. 2023. <em>Dplyr: A Grammar of Data Manipulation</em>. <a href="https://dplyr.tidyverse.org">https://dplyr.tidyverse.org</a>.
</div>
<div id="ref-R-purrr" class="csl-entry">
Wickham, Hadley, and Lionel Henry. 2023. <em>Purrr: Functional Programming Tools</em>. <a href="https://purrr.tidyverse.org/">https://purrr.tidyverse.org/</a>.
</div>
<div id="ref-R-readr" class="csl-entry">
Wickham, Hadley, Jim Hester, and Jennifer Bryan. 2024. <em>Readr: Read Rectangular Text Data</em>. <a href="https://readr.tidyverse.org">https://readr.tidyverse.org</a>.
</div>
<div id="ref-R-tidyr" class="csl-entry">
Wickham, Hadley, Davis Vaughan, and Maximilian Girlich. 2024. <em>Tidyr: Tidy Messy Data</em>. <a href="https://tidyr.tidyverse.org">https://tidyr.tidyverse.org</a>.
</div>
</div>
</div>











            </section>

          </div>
        </div>
      </div>
<a href="session-info.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
